vocabularies-gsq/site-relationships.ttl

```diff
@@ -113,9 +259,7 @@ strel:temporal a skos:Collection ;
 
 strel:wellbore a skos:Collection ;
     skos:definition "Defines the parent-child relationship of wellbores that constitute a borehole."@en ;
-    skos:member strel:parent,
-        strel:boreholeIntersect,
-        strel:injectionExtractionPair ;
+    skos:member strel:parent ;
```

Comment:
@johnmckellar, to sync with some recent changes, can you please update the wellbore collection here with the following members?
  

>   skos:member strel:parent,
>         strel:boreholeIntersect,
>         strel:injectionExtractionPair ;

Otherwise this looks good. Happy to approve once this edit is made to align with the previous update.

Reply:
@KellyVance, Before I do anything here, please clarify what is needed. I am not entirely sure what you want, as the above is already in the code.